app/classes/query/modules/conditions.rb

```diff
@@ -163,6 +171,10 @@ def add_has_notes_fields_condition(fields, *joins)
         add_joins(*joins)
       end
 
+      def add_false_condition(col)
+        @where << "#{col} = FALSE"
```

Comment:
Can I get away with:
```ruby
@where = FALSE
```
Makes things a little simpler for mySQL. But can it mess things up? I.e., is there ever a case where there are no Observations but there are results?